plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -126,5 +127,13 @@ func (a *quotaAdmission) Admit(attr admission.Attributes) (err error) {
 	if attr.GetSubresource() != "" {
 		return nil
 	}
+	initialized, err := util.IsObjInitialized(attr)
+	if err != nil {
+		return err
```

Comment:
Object count quota should still apply to uninitialized resources

Reply:
Yeah, it is an exception.

Reply:
Maybe quota needs to be in two buckets, at POST time and at initialization completion time.

Reply:
agree with @liggitt , object count quotas need to apply to uninitialized resources.  we are not setup to to support that easily today, and quota will need some refactoring.

i think we should do the following in 1.9

* adding generic object count quota 
* validation performed prior to quota charges

and gate graduation of initializers pending above completion?

Reply:
Hmm, who's going to replenish the object count if the object is deleted before initialized? The replenish controller won't see the deletion of uninitialized objects.

Reply:
I think we should drop this and let the evaluator Handles() methods decide what to handle... object count ones will stay with just Create, ones that inspect can do count on create and usage on the Uninitialized->Initialized transition

Reply:
> Hmm, who's going to replenish the object count if the object is deleted before initialized? The replenish controller won't see the deletion of uninitialized objects.

eventually, resync will correct it, but that should go on the list of things to improve in the interaction between quota and initialization

Reply:
Eh, the resync needs a [List](https://github.com/kubernetes/kubernetes/blob/master/pkg/quota/evaluator/core/pods.go#L77-L79), which don't see uninitialized objects, so it will ignore the object counts for the yet-to-initialize objects.